src/applications/static-pages/download-1095b/components/App/index.unit.spec.jsx

```diff
@@ -43,10 +43,6 @@ describe('App component', () => {
   setupAvailableFormsResponse(server, 200, goodAvailableFormsResponse);
 
   const unauthenticatedState = {
-    featureToggles: {
-      loading: false,
```

Comment:
I don't think `loading` does anything because it wasn't part of `state.featureToggles`. Perhaps this was leftover from a previous implementation.